packages/eslint-plugin-format-js-tag/src/index.ts

```diff
@@ -21,8 +21,7 @@ export const rules = {
     defaultOptions: [],
     meta: {
       docs: {
-        category: "Stylistic Issues",
-        description: "",
+        description: "Stylistic Issues",
```

Comment:
@mitchellhamilton  category property removed from docs config as of this PR https://github.com/typescript-eslint/typescript-eslint/pull/3800, replaced the description in lieu of not knowing where else to put it

Reply:
```suggestion
        description: "Formats `js`, `ts` and `tsx` template tags with Prettier",
```